prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -374,6 +390,20 @@ postsubmits:
           - prow/istio-integ-k8s-tests.sh
         nodeSelector:
           testing: test-pool
+    - name: e2e-simpleTestsMinProfile-master
+      <<: *job_template
+      optional: true
```

Comment:
is this flag needed for postsubmit test?

Reply:
Interesting, I copied it from release 1.1 file, will remove from both

Reply:
Actually I do not want to change anything for release 1.1. Just removed it for master